_posts/platform/app/2000-01-01-log-drain.md

```diff
@@ -2,54 +2,145 @@
+work when trying to find an information.
+A few SaaS solutions exist such as Logentries, Logmatic.io, Papertrail or
+Graylog. You can also host such solution on Scalingo. Here is a tutorial on
+[how to deploy the ELK
+stack]({% post_url platform/getting-started/2000-01-01-getting-started-with-elk %})
+on the platform.
+
+Scalingo also supports sending applications logs using the standard
+syslog protocol over UDP, TCP or TCP+TLS.
+
+{% note %}
+Adding a log drain to an application will not disable
+[Scalingo logs]({% post_url platform/app/2000-01-01-logs %})
+{% endnote %}
 
-## Third-party Provider integrations
+## Third-Party Provider Integrations
 
-Our system is also able to send logs to the following SaaS log processing providers:
+Our system is able to send logs to the following SaaS logs processing providers:
 
-* Scalingo hosted ELK stack ([how to setup]({% post_url platform/getting-started/2000-01-01-getting-started-with-elk %}))
-* Datadog
-* Logentries
-* Graylog
-* Papertrail
-* Syslog
+* [Scalingo hosted ELK stack](#scalingo-hosted-elk-stack)
+([how to setup]({% post_url platform/getting-started/2000-01-01-getting-started-with-elk %}))
+* [Datadog](#datadog)
+* [Logentries](#logentries)
+* [Graylog](#ovh-hosted-graylog)
+* [Papertrail](#papertrail)
+* [Syslog](#syslog-tcptls)
+
+If you would like another provider to be added, feel free to contact Scalingo
+support.
 
 ## Configuration of your Application on Scalingo
 
-As this feature is private beta, you cannot activate logs drain by yourself. You need to add an
-environment variable containing the token to the used logs management service, and contact the
-support so that we activate the logs drain. In a matter of minutes, your logs flow to the logs
-management service.
-
-The format of the environment variable depends on the provider:
-
-* Scalingo hosted ELK stack: `https://USER:TOKEN@logstash-app-name.osc-fr1.scalingo.io`
-* Datadog: `datadog://DD_API_KEY?region=REGION`. `DD_API_KEY` is the Datadog API key you can
-    find [here](https://app.datadoghq.com/account/settings#api). `REGION` value
-    is either `eu` or `us` depending on your Datadog account region.
-* Logentries: `https://webhook.logentries.com/noformat/logs/LOG-TOKEN`.
-    Instructions to get the `LOG-TOKEN` are available on [Logentries
-    documentation](https://docs.logentries.com/docs/http-post).
-* OVH hosted Graylog: `ovh://:TOKEN@REGION.logs.ovh.com:6514`. Instructions to
-    get the `TOKEN` and `REGION` variables are available on [OVH
-    documentation](https://docs.ovh.com/fr/logs-data-platform/quick-start/).
-* Papertrail: `syslog+tls://logsN.papertrailapp.com:XXXXX`. `logsN` and `XXXXX`
-    are the log host and port number shown under [log
-    destination](https://papertrailapp.com/account/destinations).
-* Syslog TCP+TLS: `tcp+tls://example.com:12345`
+The log drains configuration is done using the
+[CLI](https://doc.scalingo.com/cli).
```

Comment:
@EtienneM is it possible to use `post_url` in this case ?

Reply:
`post_url platform/cli/2000-01-01-start`?